### ISSUE TITLE
Save as resource (.tres) button request

### ISSUE THREAD

**Issue by JustTheNib** [squrious/godot-fastnoiselite-configurator-plugin#1]
I can't seem to find a way to save the parameters as a fast noise lite resources. Even when loading a new one and clicking paste properties it does not seem to take them from the clipboard. Would it be too much trouble to add a save as resource button in the tool tab?

**Comment by squrious** [comment]
Hi, thanks for your suggestion. It seems very reasonable to me, I'll take a look :)

**Comment by squrious** [comment]
@JustTheNib Can you test with [this version](https://codeload.github.com/squrious/godot-fastnoiselite-configurator-plugin/zip/refs/heads/feature/save_resource) please?

(This also moves the plugin to a dedicated window, to avoid too much tabs in the editor)

If it works fine for you I'll make a new release

**Comment by JustTheNib** [comment]
Yup it worked for me! Sorry for the late response, new school semester stole some time from me.
I struggled to find the Project > Tools menu as I have not used it before. I would recommend updating that in the readme with screenshots too.
Thank you so much!
